### DISCUSSION TITLE
Advancing DevToys localization

### DISCUSSION THREAD

**Discussion by tradespin** [DevToys-app/DevToys#776] [Q&A]
I was wondering if it would be possible to give some of the localization team on Crowdin permissions to approve translations. There are a few locales (Chinese Simp. & Trad, Cornish, French, German, Greek, Hindi, Italian, Japanese, Korean, Eur. Portuguese, Spanish, Turkish & Ukrainian) that are 100% translated, but whose translations are not approved.
These locales are ready to be integrated into DevToys, so can whoever manages the Crowdin project please give us the appropriate permissions to make sure we can finalise the projects, and hopefully see DevToys in our own languages soon?

**Comment by comradekingu** [comment]
They have coverage, but aren't finished. I don't think the difference is someone said so individually for each string (removing even the ability of others to do anything about it).
